editor/modes/visual-editor/block-list.js

```diff
@@ -220,6 +220,7 @@ class VisualEditorBlockList extends Component {
 					/>,
 					<VisualEditorSiblingInserter
 						key={ 'sibling-inserter-' + uid }
+						uid={ uid }
 						insertIndex={ index + 1 }
```

Comment:
With `uid` being passed, can (and/or should) we determine `insertIndex` in its `connect` selector?

Reply:
> can we? should we?

We can. To answer the question of _should_, I tried doing it in b3218c7. Looks OK enough; `insertIndex` defaults to 0 to account for the one place where SiblingInsert was used without an associated `uid`. That default is something you may object to. Let me know.